src/pt/mediocretoons/build.gradle

```
@@ -1,9 +1,8 @@
 ext {
     extName = 'Mediocre Toons'
     extClass = '.MediocreToons'
-    themePkg = 'greenshit'
     baseUrl = 'https://mediocretoons.com'
-    overrideVersionCode = 0
+    extVersionCode = 10
```

The current version is 6
https://github.com/keiyoushi/extensions/blob/82b79ee4509c57a24772966c3ef706433c3536d9/index.json#L23768
```suggestion
    extVersionCode = 7
```